beacon-chain/db/kv/lightclient_test.go

```diff
@@ -2,221 +2,182 @@ package kv
 
 import (
 	"context"
+	"fmt"
+	"math/rand"
 	"testing"
 
+	fieldparams "github.com/prysmaticlabs/prysm/v5/config/fieldparams"
+	"github.com/prysmaticlabs/prysm/v5/config/params"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/interfaces"
+	light_client "github.com/prysmaticlabs/prysm/v5/consensus-types/light-client"
 	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
 	enginev1 "github.com/prysmaticlabs/prysm/v5/proto/engine/v1"
-	ethpbv1 "github.com/prysmaticlabs/prysm/v5/proto/eth/v1"
-	ethpbv2 "github.com/prysmaticlabs/prysm/v5/proto/eth/v2"
+	pb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
 	"github.com/prysmaticlabs/prysm/v5/runtime/version"
 	"github.com/prysmaticlabs/prysm/v5/testing/require"
+	"github.com/prysmaticlabs/prysm/v5/time/slots"
+	"google.golang.org/protobuf/proto"
 )
 
+func createUpdate(t *testing.T, v int) (interfaces.LightClientUpdate, error) {
+	params.SetupTestConfigCleanup(t)
+	config := params.BeaconConfig()
```

Comment:
You don't need this if you don't change any values. Also from a logical point of view, this should happen in the test itself, not in a helper function.